website/blog/2023-02-28-ferretdb-release-0-9-2.md

```diff
@@ -0,0 +1,107 @@
+
+![FerretDB v0.9.1 - Minor Release](/img/blog/ferretdb_v0.9.2.jpg)
+
+<!--truncate-->
+
+The past few weeks were exciting: our cooperation with the open source community is stronger than ever.
+The feedback of developers and service providers makes us confident that we are on the right track with creating truly open source alternative to MongoDB.
+
+Before we go into all the exciting new features in this release, let’s recap FerretDB and what we’re striving to achieve.
+[FerretDB](https://www.ferretdb.io/) is an [open-source alternative to MongoDB](https://blog.ferretdb.io/5-database-alternatives-mongodb-2023/) that converts MongoDB drivers and protocols into SQL with a backend on PostgreSQL.
+
+We are gradually increasing compatibility with MongoDB– including tools, drivers, and the entire ecosystem, so FerretDB can act as a fully open source drop-in replacement for it.
+Be sure to try it out!
+
+Here are some of the notable changes in our previous releases:
+
+* Basic support for aggregation pipelines
+* More filtering query pushed to the backend for Tigris and PostgreSQL
+* Client TLS certificate validation
+* Support for raspberry pi
+* Basic telemetry service
+
+This above list is by no means exhaustive; it’s only a snippet of some of the recent additions.
+If you missed our previous releases, see the [complete list of all the updates](https://github.com/FerretDB/FerretDB/releases/) to FerretDB.
+We’ll be adding lots of new features as we approach the FerretDB GA.
+If you have a feature request,feedback, or discovered a bug, please don’t hesitate to [contact us](https://docs.ferretdb.io/#community).
+We’d love to hear from you!
+
+Here are the latest updates on this new version of FerretDB:
+
+## New Features
+
+Coming into this release is support for the `listIndexes` command.
+With this command, you can now access a collection’s indexes.
+Besides `listindexes` command, we’ve also added support for `$addToSet` update operators, meaning you can easily add elements to an array as long as the element does not previously exist.
+Let’s take a look at an example:
+
+```js
+db.states.insertOne({
+  _id: 1,
+  name: "California",
+  colors: ["blue", "gold", "red"]
+})
+```
+
+Use the `$addtoSet` operator to update the array with more elements.
+
+```js
+db.states.updateOne(
+  { _id: 1 },
+  { $addToSet: { colors: "green" } }
+)
+```
+
+Another new feature implemented in this release is the support for $pullArrray.
+This features updates an array by removing all entries of the specified element.
+:
+
+```js
+db.tested.insertOne( { _id: 1, a: [ 1, 0, 2, 3, 3, 5, 0 ] } )
+```
+
+Suppose we have the above document and want to update it using the $pullAll operator:
+
+```js
+db.tested.updateOne(
+    { _id: 1 },
+    { $pullAll: { scores: [ 0, 3 ] } }
+)
+```
+
+The collection is subsequently purged of all existing instances of the elements in the specified array.
```

Comment:
The document is effected not the collection. Unless the multi option is specified. Plus updateOne will only update a single document.